src/Microsoft.OData.Core/UriParser/Aggregation/AggregateExpressionToken.cs

```diff
@@ -6,50 +6,34 @@
 
 #if ODATA_CLIENT
 namespace Microsoft.OData.Client.ALinq.UriParser
-{
-    using Microsoft.OData.UriParser.Aggregation;
 #else
 namespace Microsoft.OData.UriParser.Aggregation
-{
 #endif
+{
+    using System.Collections.Generic;
     using Microsoft.OData.UriParser;
 
-    internal sealed class AggregateExpressionToken : QueryToken

```

Comment:
>AggregateExpressionToken [](start = 26, length = 24)

Let's call this class (and file) AggregateTransformationToken (*see aggregation spec 3.1)